provisioning/ci/prepare-aws-resources.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+cd "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+CI_PS_ACCOUNT_ID=480319613404
+PROFILE_NAME="Keboola-CI-Platform-Services-Team-AWSAdministratorAccess"
```

Comment:
A zase ty profily :smile: Musime to nekde probrat kolektivne, ale snazim se ted, kde to jde, tak je nedavat - naucili me to v SRE, ze lepsi si profil nastavit jen jednorazove v terminalu a pak pustit skript. Pokud mas ale nejaky power-tips, proc to naopak do skriptu apod. davat, jsem :ear_with_hearing_aid:

Reply:
protoze si to jednorazove nastavis v terminalu a po trech hodinach to jdes updatnout a spustis command v jinym terminalu z 15, kde mas nastavenej jinej profil